src/app/components/code/index.html

```diff
@@ -0,0 +1,39 @@
+<sky-demo-page
+  pageTitle="Code"
+  navTitle="Code"
+  skyuxModule="SkyCodeModule"
+  packageName="@blackbaud/skyux-lib-code-block"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The code component formats text as a code snippet within larger blocks of text. Within the block of text, wrap <sky-code>sky-code</sky-code> tags around the code snippet. To format longer code snippets that stand alone as separate blocks of text, use the <a routerLink="/components/code-block">code block component</a> instead.
+    </p>
+    <p>
+      If a code snippet has text within <sky-code>&lt;</sky-code> and <sky-code>&gt;</sky-code> characters, then the component automatically appends a closing tag unless the code snippet is valid HTML. To prevent this, use "&#38;lt;" to escape <sky-code>&lt;</sky-code> and "&#38;gt;" to escape <sky-code>&gt;</sky-code>. For example, escape the <sky-code>&lt;</sky-code> and <sky-code>&gt;</sky-code> characters in the code snippet <sky-code>public Task&lt;Generic> MethodName()</sky-code> or the component automatically appends <sky-code>&lt;/Generic></sky-code> to the end: <sky-code>public Task&lt;Generic> MethodName()&lt;/Generic></sky-code>.
+    </p>
+    <p>
+      <sky-alert alertType="info">
+        To use the code component, you must set up the code block plugin. To install the plugin, run <sky-code>npm install @blackbaud/skyux-builder-plugin-code-block --save --save-exact</sky-code>. Then add the plugin to your <sky-code>skyuxconfig.json</sky-code> file: <sky-code>"plugins": ["@blackbaud/skyux-builder-plugin-code-block"]</sky-code>. You must also add <sky-code>SkyCodeModule</sky-code> as an import to your <sky-code>app-sky.module.ts</sky-code> file.
```

Comment:
I think the last sentence here is wrong. It basically repeats the last sentence of the Installation section, but I think what we need is the information from step 2 of the Installation section in https://developer.blackbaud.com/stache/components/code. I _think_ that this sentence should be "You must also import `SkyCodeModule` into your `app-extras.module.ts` file and declare it as an export." ... @Blackbaud-BobbyEarl, does that sound good to you?

Reply:
We should remove the last sentence since it is duplicated in the installation section.

Our instructions used to be `app-extra.module.ts`, and that still works, but anyone using the migrate plugin and is only at least SKY UX 3 will also have an `app-sky.module.ts`, where we automatically stick any SKY UX modules used within the SPA.

Reply:
So are the instructions at https://developer.blackbaud.com/stache/components/code in step 2 wrong?

Reply:
@Blackbaud-DeniseCisneros, I talked with Bobby, and we decided that we should just delete the last sentence here and then create a separate issue to get the Stache and SKY UX versions of these docs in sync. Sorry for the confusion here.

Reply:
Thanks for tracking that one down! I'll make the edit now. Should the issue that gets created go in Stache or the SKY UX docs repo?

Reply:
I created an issue in the SKY UX docs (https://github.com/blackbaud/skyux2-docs/issues/931). Based on talking to David this week about who owns what, I think it makes sense there.